provincial data/tabula-Population_Province_Level_CBS_2018.csv

```diff
@@ -0,0 +1,47 @@
+"",,,Population,
+District and Type of Local Level,Household,,,
+"",,Total,Male,Female
+Province 1,,,,
+Total,"992,445","4,534,943","2,166,536","2,368,407"

```

Comment:
Instead of rows 1-3 as they are now, Can we have a row that specifies what each of the items in a data row represent? Are they "level, number of households, total population, male population, female population"? (That's just my guess of what they are.) That will help make it easier to understand.